docs/source/api.rst

```diff
@@ -0,0 +1,56 @@
+..
+.. Copyright (C) Arm Ltd. 2021.  ALL RIGHTS RESERVED.
+..
+.. See file LICENSE for terms.
+..
+
+API
```

Comment:
maybe "API specification"?

Reply:
Since this is not a full document, I avoided to call it specification. In the top of the page we do provide references to the full spec.

Reply:
maybe "API reference" then?

Reply:
I was thinking UCX API or just API. I'm not sure what "reference" adds to the context. Maybe "API Documentation" but IMHO documentation part is kind of obvious...

Reply:
okay, since there is no better suggestion for now let's keep it as is. can always improve later.